vendor.conf

```diff
@@ -44,13 +44,49 @@ github.com/tonistiigi/fsutil dea3a0da73aee887fc02142d995be764106ac5e2
 github.com/xeipuuv/gojsonpointer e0fe6f68307607d540ed8eac07a342c33fa1b54a
 github.com/xeipuuv/gojsonreference e02fc20de94c78484cd5ffb007f8af96be030a45
 github.com/xeipuuv/gojsonschema 93e72a773fade158921402d6a24c819b48aba29d
-golang.org/x/crypto 558b6879de74bc843225cde5686419267ff707ca
-golang.org/x/net 7dcfb8076726a3fdd9353b6b8a1f1b6be6811bd6
-golang.org/x/sync 450f422ab23cf9881c94e2db30cac0eb1b7cf80c
+# golang.org/x/crypto 558b6879de74bc843225cde5686419267ff707ca
+# golang.org/x/net 7dcfb8076726a3fdd9353b6b8a1f1b6be6811bd6
+# golang.org/x/sync 450f422ab23cf9881c94e2db30cac0eb1b7cf80c
 golang.org/x/sys 95c6576299259db960f6c5b9b69ea52422860fce
+golang.org/x/crypto 3fbbcd23f1cb824e69491a5930cfeff09b12f4d2
+# golang.org/x/net c427ad74c6d7a814201695e9ffde0c5d400a7674
+# golang.org/x/sys 8f0908ab3b2457e2e15403d3697c9ef5cb4b57a9
 golang.org/x/text f72d8390a633d5dfb0cc84043294db9f6c935756
 golang.org/x/time a4bde12657593d5e90d0533a3e4fd95e635124cb
 google.golang.org/genproto d80a6e20e776b0b17a324d0ba1ab50a39c8e8944
 google.golang.org/grpc v1.3.0
 gopkg.in/yaml.v2 4c78c975fe7c825c6d1466c42be594d1d6f3aba6
-vbom.ml/util 928aaa586d7718c70f4090ddf83f2b34c16fdc8d
+# vbom.ml/util 928aaa586d7718c70f4090ddf83f2b34c16fdc8d
+
+github.com/coreos/etcd 46ee06a85cdf0c02ea453bbae71c86f27f3f2ee5
+github.com/emicklei/go-restful ff4f55a206334ef123e4f79bbf348980da81ca46
+github.com/emicklei/go-restful-swagger12 dcef7f55730566d41eae5db10e7d6981829720f6
+github.com/ghodss/yaml 0ca9ea5df5451ffdf184b4428c902747c2c11cd7
+github.com/go-openapi/jsonpointer 46af16f9f7b149af66e5d1bd010e3574dc06de98
+github.com/go-openapi/jsonreference 13c6e3589ad90f49bd3e3bbe2c2cb3d7a4142272
+github.com/go-openapi/spec 6aced65f8501fe1217321abf0749d354824ba2ff
+github.com/go-openapi/swag 1d0bd113de87027671077d3c71eb3ac5d7dbba72
```

Comment:
I'm going to look into some of these dependencies, this still seems like a crazy number of dependencies for a client.

Reply:
I guess the k8s machinery dependencies are just bloated, so there isn't much we can do about this for now...